superset-frontend/plugins/plugin-chart-echarts/src/Bubble/constants.ts

```diff
@@ -26,6 +26,7 @@ export const DEFAULT_FORM_DATA: Partial<EchartsBubbleFormData> = {
   logYAxis: false,
   xAxisTitleMargin: 30,
   yAxisTitleMargin: 30,
+  truncateXAxis: false,
```

Comment:
Note that for bubble chart, I'm assuming truncation is disabled, as it's more likely people want to see it starting from origo in this case (having a truncated x axis but a non-truncated y axis doesn't make sense for a scatter plot IMO).

Reply:
No objection. We can cherry-pick the change for the next patch.